GDevelop.js/TestUtils/GDJSMocks.js

```diff
@@ -232,6 +288,73 @@ class RuntimeScene {
   getOnceTriggers() {
     return this._onceTriggers;
   }
+  getAsyncTasksManager() {
+    return this._asyncTasksManager;
+  }
+}
+
+/**
+ * A container for objects lists that should last more than the current frame.
+ * It automatically removes objects that were destroyed from the objects lists.
+ */
+class LongLivedObjectsList {
+
+  // TODO: This class was NOT tested yet. It was adapted from the game engine implemented.
```

Comment:
Could you add the list of things that should be done, it's not straightforward

Reply:
This is part of the work to complete the other TODO: "TODO: Add a test simulating the deletion of an object before a task is finished".
The class itself should work, but no code is using it for now (because I lacked time to do the test just mentioned).